app/src/main/kotlin/com/softteco/template/data/deviceData/ThermometerData.kt

```diff
@@ -0,0 +1,11 @@
+package com.softteco.template.data.deviceData
```

Comment:
@pcfaktor why not [Kotlin convention?](https://kotlinlang.org/docs/coding-conventions.html#naming-rules)

Reply:
Of course, this is not an important stylistic remark. Kotlin conv allows both concatenation and camel case. I believe the first one is preferable, because:

1. [Names of packages are always lowercase and do not use underscores...](https://kotlinlang.org/docs/coding-conventions.html#naming-rules) the rest of it sounds like workaround;
2. Forces you to pick a good one-word name :smiley: ;
3. Consistency across ecosystems;
4. Package import declaration in camel case looks almost the same as importing a top-level variable.